spec/models/author_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Author do
+  let(:author) { FactoryBot.create(:author) }
+
+  describe 'create author object' do
+    it 'create author' do
+      expect(author).to be_truthy
+    end
+  end
```

Comment:
What is purpose of this test?
Remove it
Add validations tests and associations tests via `shoulda_matcher`